astro/src/content/docs/get-started/download-and-install/docker.mdx

```diff
@@ -473,17 +219,23 @@ Such output indicates that the plugin has been installed and can be used.
 
 ## Accessing the Host Machine
 
-The default FusionAuth Docker configuration sets up the network using the `bridge` configuration. This means that all the hosts defined by `docker-compose` can access each other. However, it means that any applications running on your host machine cannot be accessed by FusionAuth using `localhost`.
+The default FusionAuth Docker configuration sets up the network using the `bridge` configuration. This means that all the hosts defined by `docker-compose.yml` can access each other. However, it means that any applications running on your host machine cannot be accessed by FusionAuth using `localhost`.
 
 This is typically only an issue when FusionAuth is accessing resources outside of the Docker network to, for example, send email or request a webhook. If a Rails application is running locally on a Mac and you want FusionAuth, running in Docker, to send a webhook payload to it, `localhost` won't work.
 
 In this situation, do one of the following:
 
-* Use `host.docker.internal` as the hostname; this works on macOS and Windows hosts
+* Use `host.docker.internal` as the hostname instead of `localhost`; this works on macOS and Windows hosts, for linux you have to add `--add-host=host.docker.internal:host-gateway` to the docker engine.
```

Comment:
```suggestion
* For macOS and Windows hosts, you can use `host.docker.internal` as the hostname instead of `localhost`.
* For Linux hosts, add `--add-host=host.docker.internal:host-gateway` to the docker engine.
```

Updated this for clarity, see next comment for correctness.

Reply:
Can you explain what you mean by 'docker engine' here? Also, this post indicates you can do the same with the `extra_hosts` YAML key: https://stackoverflow.com/questions/70725881/what-is-the-equivalent-of-add-host-host-docker-internalhost-gateway-in-a-comp  

That seems preferable.